test/models/account.model.test.js

```diff
@@ -22,16 +22,17 @@ const fakeAccountData = {
   tradingName: undefined
 }
 const fakeApplicationData = {
+  id: '403710b7-18b8-e711-810d-5065f38bb461',
```

Comment:
I would save a bit of typing here and just use 'APPLICATION_ID' rather than than an actual UUID.

Reply:
Changed as requested